runtime/common/src/constants.rs

```diff
@@ -20,26 +20,26 @@ pub const TOKEN_DECIMALS: u8 = 8;
 
 parameter_types! {
 	/// Clone + Debug + Eq  implementation for u32 types
-	pub const MaxDataSize: u32 = 30;
+	pub const Const30u32: u32 = 30;
```

Comment:
This doesn't seem more clear to me.  Is it always going to be 30? Shouldn't it be MaxSchemaGrants or something?

Reply:
Perhaps? The old one collided with a generic type and it wasn't very clear before either. Open to other names. MaxSchemaGrants works for me

Reply:
I second `MaxSchemaGrants`

Reply:
Moved over to #1453